packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -122,12 +167,15 @@ const Statement = ({
     if (error) return <p>{error}</p>;
 
     const filter_component = (
-        <CompositeCalendar
-            input_date_range={filtered_date_range}
-            onChange={handleDateChange}
-            from={date_from}
-            to={date_to}
-        />
+        <React.Fragment>
```

Comment:
Can we move this outside of `Statement`'s render logic? Same counts for `getAccountStatistics`, also they're both components so should be named as such, e.g. `AccountStatistics` and `FilterComponent`